### ISSUE TITLE
Auhorship

### ISSUE THREAD

**Issue by CFGrote** [kuangmeng/CIMSA#15]
General checks (cf. openjournals/joss-reviews#2672)

The submitting author has made major if not all contributions to the software. However,
the paper acknowledges contributions from other developers and researchers but these remain unnamed. Hence, the completeness question in openjournals/joss-reviews#2672 cannot be answered.

**Comment by will-rowe** [comment]
Please advise here @kuangmeng. Did anyone else contribute directly to your submission? I read the paper acknowledgement as a generic thank you to those in the field - if this is correct, maybe rephrase it to avoid confusion (or remove if you want, as it's not required due to citations being sufficient)?

**Comment by xyzmjf** [comment]
There remains ambiguity regarding the authorship. If each of the routines is a redeveloped version of a previously published algorithm, then it would help greatly to mention the literature reference for these in the code. This would allow the correctness of implementation to be examined more easily. One would hope this is not just a copy paste of code from elsewhere, I assume it is not.
